### PR TITLE
Module #7 Concurrency

### PR DESCRIPTION
**Pull request by chrismld**
Content ready for module "Concurrency"

### CODE REVIEW

File: concurrency/includes/2-channels.md [hunk 1]
```diff
@@ -0,0 +1,460 @@
+# Communication Mechanisms
```
Comment (meaghanlewis) [inline concurrency/includes/2-channels.md:1]: @christianhxc can you think of a way to split this unit up? 460 lines total will take the learner maybe 15-20 minutes to finish, and that's a little long for one unit.

Comment (cmelendezeqx) [reply concurrency/includes/2-channels.md:1]: @meaghanlewis oh right, I didn't notice the numbers of lines, let me change it and I'll let you know

Comment (chrismld) [reply concurrency/includes/2-channels.md:1]: @meaghanlewis oh right, I didn't notice the number of lines, let me change it and I'll let you know

Comment (chrismld) [reply concurrency/includes/2-channels.md:1]: @meaghanlewis done, let me know your thoughts

Comment (meaghanlewis) [reply concurrency/includes/2-channels.md:1]: thanks for splitting this file up, looks good to me!

**Review by meaghanlewis — APPROVED** [review]

**Pull request merged**